components/invitation_modal/invitation_modal_members_step/invitation_modal_members_step.jsx

```diff
@@ -118,6 +121,23 @@ class InvitationModalMembersStep extends React.PureComponent {
         this.props.onSubmit(users, emails, this.state.usersInputValue);
     }
 
+    shouldShowPickerError = () => {
+        const {userLimit, currentUsers, userIsAdmin} = this.props;
+        const admin = userIsAdmin(this.props.currentTeamId);
```

Comment:
Since `userIsAdmin` is only ever called here using a prop passed into the component, can we just calculate this in the connector instead? You have access to passed in props as part of the second argument to `mapStateToProps`

`function mapStateToProps(state: GlobalState, ownProps: OwnProps)`

Reply:
Good point.